app/services/db-export/db-export.service.js

```diff
@@ -1,28 +1,20 @@
 'use strict'
 
 /**
+ * Exports the entire db
  * @module DbExportService
  */
 
-const ConvertToCSVService = require('../../services/db-export/convert-to-csv.service')
-const CompressFilesService = require('../../services/db-export/compress-files.service')
-const DeleteFileService = require('../../services/db-export/delete-file.service')
-const ExportDataFilesService = require('../../services/db-export/export-data-files.service')
-const FetchBillingChargeCategoriesService = require('../../services/db-export/fetch-billing-charge-categories.service.js')
-const SendToS3BucketService = require('../../services/db-export/send-to-s3-bucket.service')
+const TableExportService = require('../db-export/table-export.service.js')
 
+/**
+ * Calls TableExportsService giving it a tableName and schemaName
+ */
 async function go () {
-  const data = await FetchBillingChargeCategoriesService.go()
-
-  const tableConvertedToCsv = ConvertToCSVService.go(data.headers, data.rows)
-
-  const filePath = await ExportDataFilesService.go(tableConvertedToCsv, data.tableName)
-
-  const compressedFilePath = await CompressFilesService.go(filePath)
-
-  await SendToS3BucketService.go(compressedFilePath)
+  const tableName = 'billing_charge_categories'
+  const schemaName = 'water'
 
-  await DeleteFileService.go(filePath)
+  return await TableExportService.go(tableName, schemaName)
```

Comment:
Having had a look at the `TableExportService` it doesn't appear to be returning anything explicitly. So, if that isn't returning anything there isn't really anything to return here. I'd suggest

```suggestion
  await TableExportService.go(tableName, schemaName)
```